index.html

```diff
@@ -213,18 +288,18 @@ <h2>
       </h2>
       <section id="key">
         <h3>
-          Content Encryption key
+          Key Encryption key
```

Comment:
I am a little confused -- content encryption key is referred in a few places in the doc: line 224, 370, 460. Why do we need a "Key Encryption Key" section here?

Reply:
@wanli There are two keys. The RSA key is not the CEK it is just used to securely exchange the CEK with the other party. The CEK is a symmetric key used to encrypt the content using AES 128 GCM.

Reply:
I see. So the " Encrypted Key" in the message is from the encrypted CEK. Thanks the for explantation!